WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -335,9 +340,12 @@ class MySiteViewModel
         _onNavigation.value = Event(getStatsNavigationActionForSite(site))
     }
 
-    private fun quickActionPagesClick() {
+    private fun quickActionPagesClick(activeTask: QuickStartTask?) {
         val site = requireNotNull(selectedSiteRepository.getSelectedSite())
         analyticsTrackerWrapper.track(QUICK_ACTION_PAGES_TAPPED)
+        if (activeTask == EDIT_HOMEPAGE) {
+            eventBusWrapper.postSticky(QuickStartEvent(activeTask))
+        }
```

Comment:
Maybe we could use `QuickStartRepository::requestNextStepOfTask` here instead? Then we wouldn't even need to pass the current active task as a parameter to `ListItemInteraction::create`.

Reply:
This method calls `activeTask.value = null` and I'm wondering if that's what we expect in this case since we complete the task only if the user clicks on the Homepage, not on the "Pages" item. What do you think? The difference is that click on Pages and back would not remove the focus point. I guess removing the focus point is OK though since the point is to show the user the menu item and by clicking it we're sure they've seen it. And they can also restart the task by clicking it again